cmd/nri-prometheus/config.go

```diff
@@ -109,6 +109,8 @@ func setViperDefaults(viper *viper.Viper) {
 	viper.SetDefault("insecure_skip_verify", false)
 	viper.SetDefault("standalone", true)
 	viper.SetDefault("disable_autodiscovery", false)
+	viper.SetDefault("enable_scrape_services", true)
+	viper.SetDefault("enable_scrape_endpoints", false)
```

Comment:
I'd rename these without the `enable_` prefix

```suggestion
	viper.SetDefault("scrape_services", true)
	viper.SetDefault("scrape_endpoints", false)
```

Reply:
mmm, I added it since there was just above disale_... but I can change that :)